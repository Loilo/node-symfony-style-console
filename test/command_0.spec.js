const wrapper = require('./wrapper')

test('00 Ensure has single blank line at start when using block element', wrapper(io => {
	io.caution('Lorem ipsum dolor sit amet')
}))
