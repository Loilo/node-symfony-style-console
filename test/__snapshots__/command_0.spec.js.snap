// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`00 Ensure has single blank line at start when using block element 1`] = `
"
 ! [CAUTION] Lorem ipsum dolor sit amet                                         

"
`;
